src/kafka/index.ts

```diff
@@ -1,122 +1,80 @@
 /* eslint-disable max-len */
 
 import * as _ from 'lodash';
-import { queue } from 'async';
+import { Kafka, logLevel } from 'kafkajs';
 
 import config from '../config';
 import log from '../util/log';
-
-const SHUTDOWN_DELAY = 5000;
-
-function kafkaLogger (type: string) {
```

Comment:
Have you considered plugging in the app logger like we did for with the old library? It has the benefit that you'll be able to see kafka client logs in kibana/cloudwatch if needed.

Reply:
Yup!!  Next PR is going to be for ssl config!

Also ah!  Yes I meant to plug that in, TY!